cross/zvbi/patches/06_sizeof-FTBFS.patch

```diff
@@ -0,0 +1,17 @@
+Description: Fix and FTBFS on ia64, s390x, alpha, and ppc64.
```

Comment:
From the description this patch this should only affect ppc64 and therefore be under `patches/ppc/` so it only gets applied when building such arch.

Reply:
Will do

Reply:
done